vignettes/introduction.Rmd

```diff
@@ -0,0 +1,126 @@
+
+The premise of MAIC methods is to adjust for between-trial differences in patient demographic or disease characteristics at baseline. When a common treatment comparator is unavailable, a MAIC assumes that differences between absolute outcomes that would be observed in each trial are entirely explained by imbalances in prognostic variables and effect modifiers. Under this assumption, every prognostic variable and every effect modifier that is imbalanced between the two studies must be available. This assumption is generally considered very difficult to meet. [@phillippo2016a] 
+
+There are several ways of identifying prognostic variables/effect modifiers to be used in the MAIC analyses. We can approach from a clinical perspective, seeking advice from clinical experts or looking into published paper or previous regulatory submissions. We can also approach it from a data perspective, exploring regression models or subgroup analysis to identify interactions between baseline characteristics and treatment effect.
+
+# Statistical theory behind MAIC
+
+We will briefly go over the statistical theory behind MAIC. For more detailed information, refer to Signorovitch et al. 2010. [@signorovitch2010]
+
+Let us define $t_i$ to be the treatment patient $i$ received. We assume $t_i=0$ if the patient received intervention (IPD) and $t_i=1$ if the patient received comparator treatment. The causal effect of treatment $T=0$ vs $T=1$ on the
+mean of the outcome $Y$ can be estimated as below
+
+\[
+\frac{\sum_{i=1}^{n}y_{i}(1-t_{i})w_{i}}{\sum_{i=1}^{n}(1-t_{i})w_{i}}-\bar{y}_{1}
+\]
+
+where $w_i=\frac{Pr(T_i=1\mid x_i)}{Pr(T_i=0\mid x_i)}$ is the odds that patient $i$ received treatment $T=1$ vs $T=0$ (i.e. enrolls in aggregate data study vs IPD study) given baseline characteristics $x_i$. Thus, the patients receiving $T=0$ are re-weighted to match the distribution of patients receiving $T=1$. Note that this causal effect would be the case when the outcome $Y$ is continuous. If the outcome is binary, $Y$ would be a proportion and we would use a link function such as logit to give us the causal effect in an odds ratio scale. As in propensity score methods, we may
+assume $w_i$ to follow logistic regression form
+
+\[
+w_{i}=exp(x_i^{T}\beta)
+\]
+
+However, in order to estimate $\beta$, we cannot use maximum likelihood approach because we do not have IPD for both trials. Instead, we use method of moments. We estimate $\beta$ such that the weighted averages of the covariates in the IPD exactly matches the aggregate data averages. Mathematically speaking, we want to estimate $\beta$ such that:
+
+\[
+0=\frac{\sum_{i=1}^{n}x_{i}exp(x_i^{T}\hat{\beta})}{\sum_{i=1}^{n}exp(x_i^{T}\hat{\beta})}-\bar{x}_{agg}
+\]
+
+If the $x_i$ contains all confounders and the logistic regression for $w_i$ is correctly specified, we obtain a consistent estimate of the causal effect of intervention vs comparator treatment. Above equation is equivalent to
```

Comment:
It is possible to use this estimator since a logistic regression model for the odds of receiving T = 1 vs T = 0 would, by definition, provide the correct weights for balancing the trial populations. If the $x_i$ contains all the confounders and the logistic model for $w_i$ is correctly specified, then $\hat{\theta}$ in the next equation provides a consistent estimate of the causal effect of treatment T = 0 vs T = 1 on the mean of Y among patients actually receiving treatment T = 1.

\[
\hat{\theta}=\frac{\sum_{i=1}^{n}y_{i}exp(x_i^{T}\hat{\beta})}{\sum_{i=1}^{n}exp(x_i^{T}\hat{\beta})}-\bar{y}_{agg}
\]